### DISCUSSION TITLE
How to report spam of a deleted issue?

### DISCUSSION THREAD

**Discussion by charles-cty** [community/community#121673] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have received a lot of spam emails. The spamers mentioned me and sent me spam emails. After that, they deleted the related issues, so I cannot report spam.
I have read [this](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request), but I don't know how to report spam of a deleted issue. 
One of the spam emails is:
![screenshot](https://github.com/community/community/assets/67942242/da99c65d-8384-4e09-b160-cc7e58c034b9)
[email.pdf](https://github.com/community/community/files/15165826/email.pdf)
And the [related issue](https://github.com/kaichen/asr-server-py/issues/6) is _404 Not Found_ now, so I don't know where is the spammers' account "Kyle Lee". And I think [this repo](https://github.com/kaichen/asr-server-py) itself is innocent.
So, how can I report this spam and stop this kind of spam emails?
Thanks in advance.

**Comment by excreal** [comment]
Maybe try contacting support 😁

**Comment by ebndev** [comment]
Hi @charles-cty 👋🏻 

The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
